solc/CommandLineInterface.cpp

```diff
@@ -542,7 +545,8 @@ Allowed options)",
 			"Switch to linker mode, ignoring all options apart from --libraries "
 			"and modify binaries in place."
 		)
-		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.");
+		(g_argMetadataLiteral.c_str(), "Store referenced sources are literal data in the metadata output.")
+		(g_argStandardJSON.c_str(), "Process standard JSON input / output.");
```

Comment:
What is the "JSON input/output mode"?

Shouldn't it be irrelevant to the user how it operates internally? Ideally the CLI should go through the JSON IO to reduce the number of APIs we have, but that shouldn't have any effect on the user.

The current `--standard-json` is only an interface to directly interact with the API.